client/state/ui/editor/post/actions.js

```diff
@@ -12,182 +13,190 @@ export function setEditingMode( mode, modeTitle, site ) {
 
 export function startEditingNew( site, postOptions ) {
 	debug( 'startEditingNew', site, postOptions );
-	return { type: 'TODO' };
+	return {
+		type: EDITOR_POST_CURRENT_ID,
+		siteId: site.ID,
+		postId: null
+	};
 }
 
 export function startEditingExisting( site, postID ) {
 	debug( 'startEditingExisting', site, postID );
-	return { type: 'TODO' };
+	return {
+		type: EDITOR_POST_CURRENT_ID,
+		siteId: site.ID,
+		postId: postID
+	};
 }
 
-export function toggleStickyStatus( currentStatus ) {
-	debug( 'toggleStickyStatus', currentStatus );
+export function toggleStickyStatus( siteId, postId, currentStatus ) {
```

Comment:
Well, I think it actually makes sense to have them reversed :)
The idea is to call `editPost` inside these wrappers introduced here, so the argument order is not that important.

You chose to put it last because you know that editPost has always at least 1 argument - the object, and then site and post id is kinda optional.

In these creators, there is a variable number of arguments, so I decided to put site/post at the beginning so that at least there is a consensus on whats to expect. If they were last, sometimes it would be 2nd and 3rd and sometimes 6th and 7th argument.